### PR TITLE
Feature/40 place create form

### PR DESCRIPTION
**Pull request by pawelBareja**
Please feel free to comment on this..

### CODE REVIEW

**Review by WronskiN — APPROVED** [review]

File: src/components/Place/PlaceComponents/PlaceDetails/PlaceDescription.js [hunk 1]
```diff
@@ -1,15 +1,22 @@
...
 
-const CURRENCY_NAME = "zł"
+const CURRENCY_NAME = "zł";
 
-const PlaceDescription = ({address, city, description, price, rate}) => {
+const PlaceDescription = ({ address, city, description, price, rate }) => {
+  return (
+    <div className="Card__Description">
+      <p className="Card__Item">
+        <i class="fas fa-star"></i> {rate}
```
Comment (LukasMod) [inline src/components/Place/PlaceComponents/PlaceDetails/PlaceDescription.js:9]: `<i className...`

Comment (LukasMod) [inline src/components/Place/PlaceComponents/PlaceDetails/PlaceDescription.js:9]: <i **className**=...></i>

File: src/components/PlaceCreateForm/PlaceCreateForm.js [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+
+  render() {
+    const { params, errors } = this.state;
+
+    return (
+      <div>
+        <h1>Place Create Form</h1>
+        <p>Należy wypełnić wszystkie pola</p>
+        <form
+          novalidate
```
Comment (LukasMod) [inline src/components/PlaceCreateForm/PlaceCreateForm.js:126]: noValidate

File: src/App.js [hunk 3]
```diff
@@ -23,6 +24,7 @@ function App() {
         <Switch>
           <Route path="/" exact>
             <Searchbar />
+            <PlaceCreateForm />
```
Comment (LukasMod) [inline src/App.js:27]: I propose to put it in a separate item in the menu as 'add new place' with new route

File: debug.log [hunk 4]
```diff
@@ -0,0 +1 @@
+[1129/151104.972:ERROR:directory_reader_win.cc(43)] FindFirstFile: System nie mo�e odnale�� okre�lonej �cie�ki. (0x3)
```
Comment (LukasMod) [inline debug.log:1]: ??? :)

**Review by LukasMod — APPROVED** [review]

**Pull request merged**